src/Entities/Transaction.php

```diff
@@ -526,22 +526,6 @@ public function getIsRetry()
         return $this->getAttribute('isRetry');
```

Comment:
This may be removed too.

Reply:
along with `getRetryNumber`

Reply:
These values are filled when the transaction is assigned to an invoice. In this case the retryNumber is copied from the invoice. I decide to leave it